app/main/views/suppliers.py

```diff
@@ -444,3 +443,28 @@ def create_your_account_complete():
         "suppliers/create_your_account_complete.html",
         email_address=email_address
     ), 200
+
+
+@main.route('/<framework_slug>/opportunities-overview', methods=['GET'])
```

Comment:
Thinking about this more the most natural place for this page seems to me to be in `briefs.py` and with the URL `/opportunities` - I really don't think we need the word overview in there.